aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModuleAgent.java

```diff
@@ -406,6 +408,17 @@ public void onLoadBeginSnapshot(
         }
     }
 
+    public ControlledFragmentHandler.Action onUnknownMessageSchema(
```

Comment:
This should be renamed to `onExtensionMessage`.

Reply:
renamed